### DISCUSSION TITLE
Install without docker?

### DISCUSSION THREAD

**Discussion by jayjay73** [immich-app/immich#9783] [Q&A] [answered]
Is it at all possible to install and run immich without the use of docker? I couldn't figure it out from looking around in the docs or the instll script. At some point it all seems to involve downloading some docher-compose.yaml.

There are those of us who prefer to run sour software the old fashioned UNIX way and sometimes we do so on (*gasp*) _systems where docker isn't even available_.

So, a hard dependency on docker is kind of a deal breaker.

**Comment by mmomjian** [comment, marked as answer]
It's possible. It will take a lot more because it depends on a lot of very specific system packages. You can start here if you're really inclined: https://github.com/immich-app/immich/discussions/1657 , and here: https://github.com/arter97/immich-native

These deployments are completely unsupported/community
